oci/oci_linux.go

```diff
@@ -0,0 +1,57 @@
+// +build linux
```

Comment:
Isn't this covered by the implicit build constraint from the filename?  I don't think we need this explicit constraint as well (docs [here][1]).

[1]: https://golang.org/pkg/go/build/#hdr-Build_Constraints